reservation_units/tasks.py

```diff
@@ -26,6 +29,80 @@ def _update_reservation_unit_pricings() -> None:
     logger.info(f"Updated {num_updated} reservation units with date {today}")
 
 
+@app.task(name="update_reservation_unit_pricings_tax_percentage")
+def update_reservation_unit_pricings_tax_percentage(
+    change_date: str,
+    current_tax: str,
+    future_tax: str,
+    ignored_company_codes: list[str] | tuple[str] = (),
```

Comment:
This is fine, but you can also use a suitable [abstract base class](https://docs.python.org/3/library/collections.abc.html#collections-abstract-base-classes) for the typing, e.g. `Collection` in this case.